grype/presenter/table/presenter.go

```diff
@@ -14,19 +14,27 @@ import (
 	"github.com/anchore/grype/grype/vulnerability"
 )
 
+const (
+	AppendSuppressed = "-(suppressed)"
```

Comment:
This isn't used outside this package, is it? In which case, it should not be exported (in other words, lowercase; `appendSuppressed`)

Maybe we should change this to ` (Suppressed)`? 

So output would look like: `Low (Suppressed)`?

...or lowercased, is also probably fine, `Low (suppressed)`, but probably replace the dash with a space.

Reply:
Sounds good, I can make those updates